fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/OAuthSchemaGenerator.java

```diff
@@ -102,6 +102,7 @@ protected void addCacheTable(PhysicalDataModel model) {
                 .addUniqueIndex(OAUTH20CACHE + "_" + EXPIRES, EXPIRES) // ASC is the default
                 .addPrivileges(generateGroupPrivilege())
                 .build(model);
+        cache.addTag(FhirSchemaGenerator.SCHEMA_GROUP_TAG, FhirSchemaGenerator.FHIRDATA_GROUP);
```

Comment:
should we use a different group id for the OAuth schema objects?

Reply:
I think that implied much more work were we split out the Physical Data model.  If not true, is that OK to change... @punktilious ?

Reply:
yeah, i just had a quick look in Main and we have logic like this:
```
if (dropFhirSchema || dropOauthSchema || dropJavaBatchSchema) {
            logger.info("Dropping FK constraints in the data schema: " + this.schema.getSchemaName());
            dropForeignKeyConstraints(pdm, FhirSchemaGenerator.SCHEMA_GROUP_TAG, FhirSchemaGenerator.FHIRDATA_GROUP);
        }

...

if (dropFhirSchema || dropOauthSchema || dropJavaBatchSchema) {
                        // Just drop the objects associated with the FHIRDATA schema group
                        pdm.drop(adapter, FhirSchemaGenerator.SCHEMA_GROUP_TAG, FhirSchemaGenerator.FHIRDATA_GROUP);
                    }
```

I guess I don't really understand what the "group" is supposed to signify here.

Reply:
I tried it out. It'll fail miserably. it has to be part of separating the physical model.

Reply:
it's part of the PhysicalDataModel

Reply:
Do we know anywhere else that the SCHEMA_GROUP_TAG is used?

Reply:
got it to work.... changed the tag names

Reply:
yes.  this looks exactly how i was expecting it.  marking LGTM but lets wait for robin's review to be sure we're not missing something!